netty/src/main/java/io/grpc/netty/NettyServerBuilder.java

```diff
@@ -304,9 +304,10 @@ public NettyServerBuilder maxConnectionAge(long maxConnectionAge, TimeUnit timeU
   }
 
   /**
-   * Sets a custom grace time for the graceful connection termination when the connection age
-   * exceeds the max connection age. {@code Long.MAX_VALUE} nano seconds or an unreasonably large
-   * value are considered infinite.
+   * Sets a custom grace time for the graceful connection termination after the connection age
+   * exceeds the max connection age. If the connection is still open until the grace time being
```

Comment:
The grammar looks a bit wrong there. "Connection still open" also doesn't make it obvious that is caused by streams being open. We also try to use 'calls' in this part of the API. How about:

> RPCs have the grace time to complete. RPCs that do not complete in time will be cancelled, allowing the connection to terminate.